docs/core/logging.md

```diff
@@ -178,148 +224,286 @@ Powertools for AWS Lambda (Java) extends the functionality of Log4J. Below is an
     </Configuration>
     ```
 
-You can also override log level by setting **`POWERTOOLS_LOG_LEVEL`** env var. Here is an example using AWS Serverless Application Model (SAM)
+=== "logback.xml"
 
-=== "template.yaml"
-    ``` yaml hl_lines="9 10"
-    Resources:
-        HelloWorldFunction:
-            Type: AWS::Serverless::Function
-            Properties:
-            ...
-            Runtime: java8
-            Environment:
-                Variables:
-                    POWERTOOLS_LOG_LEVEL: DEBUG
-                    POWERTOOLS_SERVICE_NAME: example
+    With logback, we leverage a custom [Encoder](https://logback.qos.ch/manual/encoders.html){target="_blank"} 
+    to provide structured logging:
+
+    ```xml hl_lines="4 5"
+    <?xml version="1.0" encoding="UTF-8"?>
+    <configuration>
+        <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+            <encoder class="software.amazon.lambda.powertools.logging.LambdaJsonEncoder">
+            </encoder>
+        </appender>
+        <logger name="com.example" level="DEBUG" additivity="false">
+            <appender-ref ref="console" />
+        </logger>
+        <root level="INFO">
+            <appender-ref ref="console" />
+        </root>
+    </configuration>
     ```
 
-You can also explicitly set a service name via **`POWERTOOLS_SERVICE_NAME`** env var. This sets **service** key that will be present across all log statements.
+## Log level
+Log level is generally configured in the `log4j2.xml` or `logback.xml`. But this level is static and needs a redeployment of the function to be changed.
+Powertools for AWS Lambda permits to change this level dynamically thanks to an environment variable `POWERTOOLS_LOG_LEVEL`.
 
-## Standard structured keys
+We support the following log levels (SLF4J levels): `TRACE` (0), `DEBUG` (10), `INFO` (20), `WARN` (30), `ERROR` (40).
```

Comment:
Do the numerical values of the enum matter?

Reply:
not sure...
```suggestion
We support the following log levels (SLF4J levels): `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`.
```